Popup/Popup.js

```diff
@@ -611,15 +612,24 @@ class Popup extends Component {
 
 		const current = Spotlight.getCurrent();
 		const containerNode = getContainerNode(this.state.containerId);
+		const lastContainerId = getLastContainer();
```

Comment:
Do we need to know the last container id? Or can we use the containerId of this Popup(`this.state.containerId`)?

Reply:
As I checked, we need to know the last container id.  For example, in the sample code lastContainerId is `panel-container5` and this.state.containerId returns `container-2`. So when we disable this.state.containerId(=`container-2`), spotlight logic still call `getContainerFocusTarget()` with `panel-container5` which will reproduce the issue.